moveit_core/collision_detection/src/world.cpp

```diff
@@ -186,6 +191,7 @@ const Eigen::Isometry3d& World::getTransform(const std::string& name, bool& fram
     }
   }
 
+  // we need a persisting isometry for the API
```

Comment:
```suggestion
  // Return an identity transform so there is a persisting isometry object for the API
```
nit

Reply:
Don't like the suggestion because we *return* a reference, not the transform.